expressEngine/test/data/success.js

```diff
@@ -0,0 +1,23 @@
+const role = {
+  supplier: 'supplier',
+  systemAdmin: 'system-admin',
+};
+
+const valid = ['Abc@gmail.com', 'abc@gmail.com', 'abcd@gmail.com', 'aBc@gmail.com', 'abC@gmail.com', 'abd@gmail.com', 'sdf@gmail.com', 'df@gmail.com', 'sdflj@gmail.com', 'ldf@gmic.com', 'd@gmail.com'];
```

Comment:
I think `validEmails` is a better option than `valid` for this variable.

Reply:
Add on to @manish-nx  **userRoles** instead of roles improves the readability